### ISSUE TITLE
Conflict with zycraft's zychorium ore-gen

### ISSUE THREAD

**Issue by frank89722** [EwyBoy/OreTweaker#11]
Forge version: 36.1.16
OreTweaker version: 2.0.1
ZYcraft: https://github.com/nikita488/ZYCraft

All configs are default

Before OreTweaker installed:
![image](https://user-images.githubusercontent.com/10683361/117677437-097ee080-b1e1-11eb-9b0d-009affab8945.png)

After:
![image](https://user-images.githubusercontent.com/10683361/117677521-1dc2dd80-b1e1-11eb-95c8-c64aba075d9c.png)

**Comment by EwyBoy** [comment]
Thanks, will look into it.

**Comment by frank89722** [comment]
It seems like only happened when replacing "minecraft:lapis_ore" in the json file, but this is only valid in only vanilla ores in the config.

I have tried to remove lapis ore in the setting for my large modpack with other moded ores in it, but it still not work.

Hope this information is helpful to you!

**Comment by EwyBoy** [comment]
Yes, had the same bug for Create. Still struggling how to fix it, but I can hard code a temporary workaround

**Comment by frank89722** [comment]
Maybe could move the adding event to high priority, and split the removing part to a low priority event? 🤔 

Also, quark seems like having the same issue. If you decide to code a temporary workaround, it would be nice to having quark support. 

All the best!

**Comment by EwyBoy** [comment]
![2021-05-13_08 58 55](https://user-images.githubusercontent.com/5883716/118090510-f2e7ad80-b3c9-11eb-9a29-9a6f7df96cde.png)
I fixed it, found the issue.

https://github.com/EwyBoy/OreTweaker/blob/ec5e21029fcaa5fe9ca2a860d0be719f5c117c0d/src/main/java/com/ewyboy/oretweaker/tweaking/construction/OreDeconstruction.java#L45

There is a loop grabbing the ore from the ore gen feature, and I check if the block != null. If yes I remove that feature. But the block is never set back to null at the start / end of the loop and will therefore remove all the next entries. 
I lost sleep over this simple little issue and took me days to debug..

I moved the `Block targetBlock = null;` inside the loop and now everything works fine.
